Index: kcm-plasma-services/kcm_plasmaservices.desktop
===================================================================
--- kcm-plasma-services.orig/kcm_plasmaservices.desktop
+++ kcm-plasma-services/kcm_plasmaservices.desktop
@@ -6,8 +6,9 @@ Categories=Qt;KDE;X-KDE-settings-system;
 X-KDE-ServiceTypes=KCModule
 X-KDE-Library=kcm_plasmaservices
 X-KDE-ParentApp=kcontrol
-X-KDE-System-Settings-Parent-Category=session
-X-KDE-Keywords=startup
+X-KDE-System-Settings-Parent-Category=advanced
+X-KDE-Keywords=startup,service,services,plasma
+X-KDE-Weight=20
 Name=Plasma Services
 Name[de]=Plasma Dienste
 Comment=Manage the main plasma services
